Author: Michael R. Crusoe <crusoe@debian.org>
Description: Remove use of Poetry 1.2 features

Poetry v1.2 is not yet in Debian
--- pytest-httpserver.orig/pyproject.toml
+++ pytest-httpserver/pyproject.toml
@@ -34,45 +34,6 @@
 [tool.poetry.urls]
 "Bug Tracker" = "https://github.com/csernazs/pytest-httpserver/issues"
 
-[tool.poetry.group.develop]
-optional = true
-
-[tool.poetry.group.develop.dependencies]
-pre-commit = "^2.20.0"
-requests = "^2.28.1"
-Sphinx = "^5.1.1"
-sphinx-rtd-theme = "^1.0.0"
-reno = "^3.5.0"
-mypy = "^0.971"
-types-requests = "^2.28.9"
-pytest = "^7.1.3"
-pytest-cov = "^3.0.0"
-coverage = "^6.4.4"
-ipdb = "^0.13.9"
-types-toml = "^0.10.8"
-
-
-[tool.poetry.group.doc]
-optional = true
-
-[tool.poetry.group.doc.dependencies]
-Sphinx = "^5.1.1"
-sphinx-rtd-theme = "^1.0.0"
-
-
-[tool.poetry.group.test]
-optional = true
-
-[tool.poetry.group.test.dependencies]
-pytest = "^7.1.3"
-pytest-cov = "^3.0.0"
-coverage = "^6.4.4"
-requests = "^2.28.1"
-mypy = "^0.971"
-types-requests = "^2.28.9"
-pre-commit = "^2.20.0"
-types-toml = "^0.10.8"
-
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
